Header header

uint16[] wrtAnchors 
float32[2] curr_pos
